ci/README.md

```diff
@@ -1,12 +1,12 @@
 ## Overview
 GitHub Actions is enabled in this repository so each pull request is checked before being allowed to merge.
-The system is based on `fiware/orion-ci:deb` which is built from master branch each time a new PR lands in master,
+The system is based on `telefonicaiot/fiware-orion:ci` which is built from master branch each time a new PR lands in master,
 providing a clean environment with all build dependencies onboard. The Dockerfile used to build this docker is available
 in the `ci/deb` directory.
 
-Note that `fiware/orion-ci:deb` is *not* rebuilt due to changes in the PR branch under test. Thus, if you are developing
+Note that `telefonicaiot/fiware-orion:ci` is *not* rebuilt due to changes in the PR branch under test. Thus, if you are developing
 a functionality that requires a new library or base system you need to do *first* a PR adding such library or base system
-to `ci/deb/build-dep.sh` and/or `Dockerfile`. Once that PR gets merged into master and `fiware/orion-ci:deb` gets rebuild
+to `ci/deb/build-dep.sh` and/or `Dockerfile`. Once that PR gets merged into master and `telefonicaiot/fiware-orion:ci` gets rebuild
 (checking progress in Dockerhub at: https://hub.docker.com/r/fiware/orion-ci/builds) your PR branch with the new 
```

Comment:
Link should be updated

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4391/commits/7b8dd0f82d9e26628b262298c07d718e9af3f5c9